src/indicator/algorithm/bollingerBands.js

```diff
@@ -1,15 +1,15 @@
-import calculator from './calculator/bollingerBands';
-import d3 from 'd3';
+import { bollingerBands as _bollingerBands } from 'd3fc-technical-indicator';
```

Comment:
Is it worth sticking with `calculator`? I wouldn't object to the naming if it was new, it just feels like we're changing it for the sake of changing it.